presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -212,25 +212,15 @@ static QueryStateMachine beginWithTicker(
             Ticker ticker,
             Metadata metadata)
     {
-        session.getTransactionId().ifPresent(transactionControl ? transactionManager::trySetActive : transactionManager::checkAndSetActive);
```

Comment:
I think we can get rid of `transactionControl` now, here and all the way up, including in `DataDefinitionTask`.

Reply:
@electrum `transactionControl` is used on the next line of this code: `if (!session.getTransactionId().isPresent() && !transactionControl) {`